packages/icons/src/manual-focus.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+  <path d="M5,15H3v4a2,2,0,0,0,2,2H9V19H5ZM5,5H9V3H5A2,2,0,0,0,3,5V9H5ZM19,3H15V5h4V9h2V5A2,2,0,0,0,19,3Zm0,16H15v2h4a2,2,0,0,0,2-2V15H19Z"/>
+  <path d="M8,16V8h2.58L12,13.37,13.42,8H16v8H14V9.8L12.76,16H11.23L10,9.8V16Z" />
```

Comment:
Could you combine paths to one path? Guideline is [here](https://github.com/AxisCommunications/practical-react-components/blob/main/DEVELOPMENT.md#icons)

Reply:
Done